### ISSUE TITLE
Add name of user or subject on the Messages Screen

### ISSUE THREAD

**Issue by vinayvenu** [avniproject/avni-webapp#925]

**Comment by mahalakshme** [comment]
It is not possible to avoid SubjectSearchService.searchByUuid and use the subject from the search results? I tried thinking about it, I feel we can do it with context providers, by setting subject value. Can you try this?

**Comment by 1t5j0y** [comment]
If we want to support browser refresh on the page, we won't have search results to refer to right?

**Comment by mahalakshme** [comment]
Yeah I felt we can do the call in MessagesView instead of SubjectsTab/UsersTab when we dont have the receiver in the context, and use the receiver in the context when we navigate from search results.

**Comment by mahalakshme** [comment]
![Screenshot 2023-03-17 at 11.58.50 AM.png](https://images.zenhubusercontent.com/62ff301d2acde40fdafa4566/1ef55632-ce33-477b-bbc8-543c82c0785e) - Also found this UI issue in staging. Adding to this card, so that the other card can be tested.

**Comment by himeshr** [comment]
Fixed the display issue seen in above screenshot as part of #928.

**Comment by AchalaBelokar** [comment]
- [ ] Not able to see any user in the message  board
showing this the Unexpected error occurred.

![Screenshot 2023-03-22 at 2.49.10 PM.png](https://images.zenhubusercontent.com/62722c06dedab50580ce62fa/adc8b01e-a291-49b9-b426-54225f5d4ff3)

**Comment by himeshr** [comment]
@AchalaBelokar the backend changes to support the Users search had not been deployed. I've triggered their deploy, it should start working soon.

**Comment by himeshr** [comment]
@AchalaBelokar Fixed the issue with loading the user sent and scheduled messages.. Please test now.
